bench/workgen/workgen.cxx

```diff
@@ -415,31 +415,16 @@ Monitor::~Monitor() {}
 int Monitor::run() {
     struct timespec t;
     struct tm *tm, _tm;
-    char time_buf[64], version[100];
+    char version[100];
     Stats prev_totals;
     WorkloadOptions *options = &_wrunner._workload->options;
     uint64_t latency_max = (uint64_t)options->max_latency * THOUSAND;
-    size_t buf_size;
-    bool first;
+    bool first_iteration;
 
-    (*_out) << "#time,"
-            << "totalsec,"
-            << "read ops per second,"
-            << "insert ops per second,"
-            << "update ops per second,"
-            << "checkpoints,"
-            << "read average latency(uS),"
-            << "read minimum latency(uS),"
-            << "read maximum latency(uS),"
-            << "insert average latency(uS),"
-            << "insert min latency(uS),"
-            << "insert maximum latency(uS),"
-            << "update average latency(uS),"
-            << "update min latency(uS),"
-            << "update maximum latency(uS)"
-            << std::endl;
+    // Format header of the table in _out stream.
+    _format_out_header();
```

Comment:
We check that _json is not NULL line 438, should we check _out is not NULL for this one too ?

Reply:
Good point. There was no such check in the original implementation but I'll add it.